brayns/Brayns.cpp

```diff
@@ -269,6 +271,38 @@ struct Brayns::Impl : public PluginAPI
         _engine->getScene().commitLights();
     }
 
+    void _createFrameBuffer()
+    {
+        if (!_engine->getFrameBuffers().empty())
+            return;
+
+        const auto& ap = _parametersManager.getApplicationParameters();
+        const auto frameSize = ap.getWindowSize();
+
+        if (ap.isStereo())
+        {
+            auto leftBuffer =
+                _engine->createFrameBuffer("0L", frameSize,
+                                           FrameBufferFormat::rgba_i8);
+            _engine->addFrameBuffer(leftBuffer);
+            _frameBuffers.push_back(leftBuffer);
+
+            auto rightBuffer =
+                _engine->createFrameBuffer("0R", frameSize,
+                                           FrameBufferFormat::rgba_i8);
+            _engine->addFrameBuffer(rightBuffer);
+            _frameBuffers.push_back(rightBuffer);
+        }
+        else
+        {
+            auto frameBuffer =
+                _engine->createFrameBuffer("default", frameSize,
+                                           FrameBufferFormat::rgba_i8);
+            _engine->addFrameBuffer(frameBuffer);
+            _frameBuffers.push_back(frameBuffer);
```

Comment:
3 times the same code, make a small helper function:
_createFrameBuffer(const std::string& name)